extensions/positron-python/src/test/positron/webAppCommands.unit.test.ts

```diff
@@ -59,17 +59,19 @@ suite('Web app commands', () => {
 
         // Stub `vscode.commands.registerCommand` to record registered command callbacks.
         vscode.commands.registerCommand = (command, callback) => {
-            assert(!commands.has(command), `Command registered more than once: ${command}`);
+            assert.ok(!commands.has(command), `Command registered more than once: ${command}`);
             commands.set(command, callback);
             return { dispose: () => undefined };
         };
 
-        // Stub `vscode.workspace.asRelativePath`.
-        sinon.stub(vscode, 'workspace').get(() => ({
-            asRelativePath(fsPath: string) {
-                return path.relative(workspacePath, fsPath);
-            },
-        }));
+        // Create a mock object for `asRelativePath`
+        const mockAsRelativePath = sinon.mock().callsFake((pathOrUri: string | vscode.Uri) => {
+            const fsPath = typeof pathOrUri === 'string' ? pathOrUri : pathOrUri.fsPath;
+            return path.relative(workspacePath, fsPath);
+        });
+        // TODO @isabelizimm: HACK HACK HACK
+        // Replace the original `asRelativePath` method with the mock object
+        vscode.workspace.asRelativePath = mockAsRelativePath;
```

Comment:
A nit style suggestion to match the stubbing style above:

```suggestion
        // Stub `vscode.workspace.asRelativePath`.
        vscode.workspace.asRelativePath = ((pathOrUri: string | vscode.Uri) => {
            const fsPath = typeof pathOrUri === 'string' ? pathOrUri : pathOrUri.fsPath;
            return path.relative(workspacePath, fsPath);
        });
```